pydarn/plotting/power.py

```diff
@@ -0,0 +1,208 @@
+# Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Cooper Ross Robertson, Summer Student 2020
+import matplotlib.pyplot as plt
+import numpy as np
+
+from typing import List
+
+import pydarn
+
+
+class Power():
+    """
+    Power class to plot SuperDARN RAWACF data power related.
+
+
+    Methods
+    -------
+    plot_interference
+    """
+    def _str_(self):
+
+        return "This class provides the following methods: \n"\
+                " - plot_interference()"
+
+    def plot_interference(cls, records: List[dict], beam_num: int,
```

Comment:
Rename this method? It plots lag0 power vs. time, which helps you visualize interference, but there's no specific interference thresholding/detection in here right?

Reply:
Thanks Ashton. The intended usage is to plot Lag0 power corresponding to politescan but I agree that a better name would be helpful. I will think of a more specific name for this method.

Reply:
@asreimer ideas on a better name? I switched it to `plot_lag0` but feel that is too specific?